src/SkyApm.Core/Tracing/SegmentContextFactory.cs

```diff
@@ -228,7 +228,7 @@ private SegmentContext GetParentSegmentContext(SpanType spanType)
                 case SpanType.Entry:
                     return null;
                 case SpanType.Local:
-                    return _entrySegmentContextAccessor.Context;
+                    //return _entrySegmentContextAccessor.Context;
```

Comment:
This line of code can be removed directly

Reply:
removed